images/linux/scripts/installers/aws-sm.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Do we really need a separate file? Can we use aws.sh script?

Reply:
Moved to aws.sh